src/StringTools/InternableString.Simple.cs

```diff
@@ -51,7 +51,7 @@ namespace Microsoft.NET.StringTools
             /// </summary>
             private int _charIndex;
 
-            public Enumerator(ref InternableString spanBuilder)
+            public Enumerator(scoped ref InternableString spanBuilder)
```

Comment:
Link to JaredPar's explanation on this since it's a workaround for a compiler bug.